### DISCUSSION TITLE
Sticky Message

### DISCUSSION THREAD

**Discussion by Triskelen** [BlackWolfWoof/yagpdb-cc#63] [General]
Is it possible to have multiple sticky messages running on the same server?

**Comment by BlackWolfWoof** [comment]
v1, most definitely
v2, also possible but you need to tweak a bit more stuff

**Comment by Triskelen** [comment]
and how would i do this ? it send the same sticky message on all chanels

**Reply by BlackWolfWoof** [reply]
Restrict them to channels

**Reply by Triskelen** [reply]
i activated whitelist and have them each 1 channel there allowed to broadcast in it didn't work as planned

**Reply by BlackWolfWoof** [reply]
If you join the official yagpdb discord support server, ping me and give me your server ID i can look at what you setup and see if there are any issues with the config

**Reply by Triskelen** [reply]
i got it running just told them both the blacklist each others channel and now it's working
